specs/AgriFood/AgriCrop/example.jsonld

```diff
@@ -0,0 +1,76 @@
+{
+    "@context": [
+        "https://forge.etsi.org/gitlab/NGSI-LD/NGSI-LD/raw/master/coreContext/ngsi-ld-core-context.json",
+        "https://raw.githubusercontent.com/GSMADeveloper/NGSI-LD-Entities/master/examples/Agri-Crop-context.jsonld"
+    ],
+    "id": "urn:ngsi-ld:AgriCrop:df72dc57-1eb9-42a3-88a9-8647ecc954b4",
+    "type": "AgriCrop",
+    "createdAt": "2017-01-01T01:20:00Z",
+    "modifiedAt": "2017-05-04T12:30:00Z",
+    "source": "https://source.example.com",
```

Comment:
source, dataProvider and entityVersion not yet adopted by ETSI, so we can remove them

Reply:
Fixed in (https://github.com/Ficodes/dataModels/commit/871e5e062edb651fa2605a30b31f96b6cc0f5960)